src/libraries/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -555,4 +555,7 @@
   <data name="net_quic_streamaborted" xml:space="preserve">
     <value>Stream aborted by peer ({0}).</value>
   </data>
+  <data name="net_http_max_active_streams_number_reached" xml:space="preserve">
+    <value>Reached the maximum number of streams on the HTTP/2 connection.</value>
```

Comment:
In what situation will this ever been seen?

Reply:
When we reach the maximum of active streams on the given HttpConnection and MaxHttp2ConnectionsPerServer > 1. Please, see [this line](https://github.com/dotnet/runtime/pull/38748/files#diff-441a1252f436e5e5b3ad0cc10b2256efR1175).

Reply:
Sure, but that exception actually makes it out to the caller?  That's not an acceptable behavior.  If we hit the max number of connections and the max number of streams on every connection, the request needs to queue, just as it does today.

Reply:
No. It **does not** go to the caller, but I think we still need to have a message on it for debugging. No?

Reply:
> No. It does not go to the caller

Ok, good, thanks.

> I think we still need to have a message on it for debugging. No?

We don't currently for this exception used to indicate a retry:
https://github.com/dotnet/runtime/blob/87919a0462bc2ebd5c256e5ad7649ff98c858e2c/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionBase.cs#L76-L84
I'm not sure why this one differs.

Reply:
This was instigated by me. I didn't realize that the exception is used only internally and saw it had hard-coded message, so I asked Alexander to move it to resources.
@alnikola I'm sorry for misleading here.

Reply:
Message is removed.